### PR TITLE
Add registry feature in GitLab catalog

### PR DESCRIPTION
**Pull request by mittz**
Added the below changed in GitLab catalog.

* Enable registry feature.
* Enable to use persistent storage.
* Optimize input parameter.

### CODE REVIEW

File: templates/gitlab/0/rancher-compose.yml [hunk 1]
```diff
@@ -13,34 +13,49 @@
       required: true
```
Comment (rawmind0) [inline templates/gitlab/0/rancher-compose.yml]: Please, specify a concrete version, instead of latest..

**Comment by rawmind0** [comment]
Hi @mittz ... Thanks so much for the PR and to contribute....

Please, take a look at the requested changes... Please, specify a concrete version in config.yml, docker-compose and rancher-compose files and docker versions, instead of latest..Latest not reproducible and breaks update rancher catalog feature...

**Comment by mittz** [comment]
Hi, @rawmind0 ... Thank you for your comment to my PR!
I fix their files for specifying a concrete docker image version.

File: templates/gitlab/0/rancher-compose.yml (hunk 1 above)
Comment (mittz) [inline templates/gitlab/0/rancher-compose.yml]: Sorry for be late to reply your comment.
I fix it, so could you review it?

**Comment by rawmind0** [comment]
Hi @mittz ...Thanks for the update.... 

LGTM...

**Pull request merged**